tests/ExampleApps/HeadlessLibAndApp/libHeadless/CMakeLists.txt

```
@@ -0,0 +1,95 @@
+cmake_minimum_required(VERSION 3.13)
+if (APPLE)
+    cmake_policy(SET CMP0042 NEW)
+endif (APPLE)
+
+project(RadiumHeadless VERSION 1.0.0)
+
+set(CMAKE_DISABLE_SOURCE_CHANGES ON)
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+if (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+    set(CMAKE_INSTALL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/installed-${CMAKE_CXX_COMPILER_ID}" CACHE PATH
+        "Install path prefix, prepended onto install directories." FORCE)
+    message("Set install prefix to ${CMAKE_INSTALL_PREFIX} for project ${PROJECT_NAME}.")
+    set(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT False)
+endif ()
+
+#-----------------------------------------------------------------------------------
+#       lib headless configuration
+#-----------------------------------------------------------------------------------
+set(libName Headless)
+set(namespaceName Radium)
+message(STATUS "Configuring library ${namespaceName}::${libName}")
+
+# Radium components : IO is not mandatory and should be removed once verified
+if (NOT IN_RADIUM_BUILD_TREE)
+    find_package(Radium REQUIRED Core Engine IO)
+    set(DEFAULT_COMPILE_OPTIONS)
+endif()
+
+find_package(glfw3 3.3 REQUIRED)
```

```suggestion
find_package(glfw3 3.2 REQUIRED)
```
Version 3.2 is shipped with Ubuntu 18.06 LTS and works well with this demo.